File: 2125. Number of Laser Beams in a Bank.cpp
class Solution {
public:
    int numberOfBeams(vector<string>& bank) {
        vector<int> ans;
        int res=0;
        for(int i=0;i<bank.size();++i){
            int k=0;
            for(int j=0;j<bank[i].size();++j){
                if(bank[i][j]=='1'){
                    k+=1;
                }
            }
            if(k>0){
                ans.push_back(k);
            }
        }
        if(ans.size()<=1){  
            return 0;
        }
        for(int el=0;el<ans.size()-1;++el){
            res+=ans[el]*ans[el+1];
        }
        return res;
    }
};

File: 2125. Number of Laser Beams in a Bank.py
class Solution:
    def numberOfBeams(self, bank: List[str]) -> int:
        ans=[]
        res=0
        for i in range(len(bank)):
            k=0
            for j in range(len(bank[i])):
               k+=int(bank[i][j])
            if k>0:
                ans.append(k)
        for i in range(len(ans)-1):
            res+=ans[i]*ans[i+1]
        return res
